packages/astro/types.d.ts

```diff
@@ -7,3 +7,9 @@ declare module 'tutorialkit:store' {
 declare module 'tutorialkit:core' {
   export const webcontainer: import('@webcontainer/api').WebContainer;
 }
+
+declare module 'tutorialkit:override-components' {
+  const topBar: typeof import('./src/default/components/TopBar.astro').default;
+
+  export { topBar as TopBar };
+}
```

Comment:
This should be moved in `env-default.d.ts`. I don't think we mean to make that visible.